src/FetchLoader.js

```diff
@@ -32,9 +31,14 @@ class FetchLoader extends React.Component {
       error ?
         <GeneSearchForm {...data} {...this.props} speciesSelectStatusMessage={`${error.name}: ${error.message}`}/> :
       loading ?
-        <GeneSearchForm {...data} {...this.props} speciesSelectStatusMessage={`Fetching species…`}/> :
+        <div>
```

Comment:
You don’t need to enclose React components in `<div>` elements unless you’re styling it (which would also suggest that you can add those styles or classes to the React element itself(.